src/main/java/com/google/sps/scheduler/Scheduler.java

```diff
@@ -46,7 +46,17 @@
    * For each task that can be scheduled, the due time is set and a list of all scheduled tasks
    * is returned.
    */
-  public static List<Task> schedule(
+  public static List<Task> scheduleInRange(List<Event> calendarEvents, List <Task> tasks, String timeZone, LocalDate startDate, LocalDate endDate){
+    LocalDate scheduleDate = startDate;
+    List<Task> scheduledTasks = new ArrayList<>();
+    while(0 >= scheduleDate.compareTo(endDate)){
```

Comment:
There is isAfter and isBefore as well: https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html#isAfter-java.time.chrono.ChronoLocalDate-